dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/resources/docker/file-manage/docker-compose.yaml

```diff
@@ -55,7 +55,7 @@ services:
     healthcheck:
       test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
       interval: 5s
-      timeout: 300s
+      timeout: 5s
```

Comment:
why we change this? to let others server wait less time?

Reply:
This is just the timeout for each retry, reducing it can improve the efficiency of the check.

https://docs.docker.com/engine/reference/builder/#healthcheck